xwiki-platform-tools/xwiki-platform-tool-packager-plugin/src/main/java/org/xwiki/tool/packager/PackageMojo.java

```diff
@@ -367,6 +367,15 @@ private void generateConfigurationFiles(File configurationFileTargetDirectory) t
                 if (entry.getName().endsWith(".vm")) {
                     String fileName = entry.getName().replace(".vm", "");
                     File outputFile = new File(configurationFileTargetDirectory, fileName);
+
+                    // Validates that the outFile location is located inside the configuration directory.
```

Comment:
s/outFile/outputFile

Reply:
ok I've now read https://sonarcloud.io/organizations/xwiki/rules?open=javasecurity%3AS6096&rule_key=javasecurity%3AS6096  and I don't think it applies to us since we control the JAR input. I would remove the unnecessary check which adds code, requires explanation as to why we need it, and costs execution time. Instead I would just add a comment to explain why we don't need a zip slip check, and add an ignore annotation (https://docs.codescan.io/hc/en-us/articles/360012109711-Ignoring-violations)

Reply:
ok, I'll update the PR accordingly.